### PR TITLE
SOCP-520 implement subscribe with groupid, add validation and test

### PR DESCRIPTION
**Pull request by fadhillahentino**
- Implement subscribe with groupID
- Add validation for pub sub object
- Add unit and integration test
- Fix docker compose
- Add field sessionID

Please help me to review
cc : @thoriqsatriya @banukusuma @rjjatson @desniaak @purbopanambang @rafi-isakh @Hvlcrs

### CODE REVIEW

**Comment by anggorodewanto** [comment]
Travis build failed @fadhillahentino

File: validation.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+// This is licensed software from AccelByte Inc, for limitations
+// and restrictions contact your company contract manager.
+
+package eventstream
+
+import (
+	"errors"
+
+	"github.com/sirupsen/logrus"
+
```
Comment (anggorodewanto) [inline validation.go]: unnecessary whitespace here

Comment (fadhillahentino) [reply validation.go]: sure, will update it
thanks
